src/index.json

```diff
@@ -10312,6 +10312,51 @@
                 "sha256Digest": "30449391c1d1e9d67210d0ef81109a11a0f74a24a6f07644c17aed58cf056173"
             }
         ],
+        "ml": [
+            {
+                "downloadUrl": "https://azuremlsdktestpypi.blob.core.windows.net/wheels/sdk-cli-v2/ml-0.0.42-py3-none-any.whl",
+                "filename": "ml-0.0.42-py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
+                    "azext.minCliCoreVersion": "2.15.0",
+                    "classifiers": [
+                        "Development Status :: 3 - Alpha",
+                        "Intended Audience :: Developers",
+                        "Natural Language :: English",
+                        "Operating System :: OS Independent",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "run_requires": [
+                        {
+                            "requires": [
+                                "pyyaml (>=5.1.0)",
+                                "azure-identity",
+                                "msrest (>=0.6.18)",
+                                "azure-core (<2.0.0,>=1.8.0)",
+                                "azure-mgmt-core (<2.0.0,>=1.2.0)",
+                                "marshmallow (<4.0.0,>=3.5)",
+                                "tqdm",
+                                "azure-storage-blob (<=12.5.0,>12.0.0b4)",
+                                "pydash (<=4.9.0)",
+                                "azure-storage-file-share (==12.3.0)",
+                                "pathspec (==0.8.*)",
+                                "cryptography (<=3.3.2)"
```

Comment:
Can you remove `pyyaml`, `msrest`, `azure-core` and `azure-mgmt-core`? They're already in the dependency list of `azure-cli`. 

It is also recommended to replace `azure-storage-blob` and `azure-storage-file-share` with `azure-multiapi-storage` installed by `azure-cli`. You can refer to this PR: https://github.com/Azure/azure-cli/pull/13414. @Juliehzl can help if you have any questions about storage dependency usages.

Reply:
I removed these dependencies in index.json. Do I also need to remove them in `setup.py`? Will ping @Juliehzl for azure-storage-blob dependency.

Reply:
You need to remove them in setup.py and regenerate the wheel and update index.json.

Reply:
After discussed with @Mossaka, the storage sdk is required by azure-ml package, not cli extension. If there is no packaging issue, I am fine with it.

Reply:
@Juliehzl LMK if there is no packaging issue.

Reply:
Removed dependencies in the CLI wheel @fengzhou-msft